.github/workflows/dev-build.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Greptime Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: GreptimeDB Operator Development Build
+
+on:
+  workflow_dispatch: # Allows you to run this workflow manually.
+    inputs:
+      repository:
+        description: The public repository to build
+        required: false
+        default: GreptimeTeam/greptimedb
```

Comment:
suggestion：No need input repository ?

Reply:
It's easy for some tests(port from https://github.com/GreptimeTeam/greptimedb/blob/main/.github/workflows/dev-build.yml#L4-L10

Reply:
Add the checkout logic in the steps.